### ISSUE TITLE
Build CI action frequently fails on csv generation

### ISSUE THREAD

**Issue by travisn** [rook/rook#11746]
**Is this a bug report or feature request?**
* Bug Report

**Deviation from expected behavior:**
The build CI action (runs only with PRs) intermittently fails when generating the csv, either with go 1.19 or 1.20, it is not correlated. 
See a failure [here](https://github.com/rook/rook/actions/runs/4239678960/jobs/7367955711). It seems to happen on one of those linux builds in over half of the PR builds.

```
Building a ClusterServiceVersion without an existing base
Bundle manifests generated successfully in ../../build/csv/ceph/amd64
Generating bundle metadata
Building a ClusterServiceVersion without an existing base
Bundle manifests generated successfully in ../../build/csv/ceph/arm64
Generating bundle metadata
sha256:f301cc503cadfd188c101354f8c5e884be1fd9881cd8dc2a80c9c9beb6682be1
Error: Process completed with exit code 1.
```

**Expected behavior:**
Successful builds

**Comment by nikzayn** [comment]
@travisn I would like to take this up. Can I get more insights on the same issue?

**Comment by travisn** [comment]
@subhamkrai is already planning on this, but you could search [good first issues](https://github.com/rook/rook/issues?q=is%3Aopen+is%3Aissue+label%3Agood-first-issue).

**Comment by travisn** [comment]
This seems to be fixed with #11906.
